packages/web/src/components/result/addons/Pagination.js

```diff
@@ -42,6 +42,8 @@ export default function Pagination(props) {
 					className={className}
 					primary={primary}
 					key={i - 1}
+					tabIndex="0"
+					onKeyPress={event => handleA11yAction(event, props.setPage(i - 1))}
```

Comment:
When passing a callback, it should be a function so the execution can be delayed. For example here the second parameter should have been a function i.e. `() => props.setPage(...)`